src/pages/en/guides/markdown-content.md

```diff
@@ -178,32 +177,80 @@ You can import components into your Markdown file with `setup` and use them alon
 ```markdown
 ---
 layout: ../layouts/BaseLayout.astro
-setup: | 
+setup: |
   import Author from '../../components/Author.astro'
   import Biography from '../components/Biography.jsx'
 author: Leon
 ---
+
 <Author name={frontmatter.author}/>
 <Biography client:visible>
   {frontmatter.author} lives in Toronto, Canada and enjoys photography.
 </Biography>
 ```
 
-<!-- 
-
 ## Importing Markdown
 
-https://github.com/withastro/rfcs/pull/147
+You can import Markdown files directly into your Astro files! You can import one specific page with `import` or multiple with `Astro.glob()`
+
+```astro
+---
+// You can use `import ... from`
+import * as greatPost from '../pages/post/great-post.md';
+
+// Or even `await import()`
+const aboutUs = await import('../pages/internal/about-us.md');
+
+// Lastly, you can import multiple files with Astro.glob
+const posts = await Astro.glob<Frontmatter>('../pages/post/*.md');
```

Comment:
You can use `Astro.glob` without a type argument right? What do you think about presenting this first usage without it for non-Typescript users given that you cover passing a `Frontmatter` type below?

```suggestion
const posts = await Astro.glob('../pages/post/*.md');
```